paderbox/array/interval/core.py

```diff
@@ -359,33 +359,39 @@ def _intervals_as_str(self):
         return i_str
 
     @property
-    def as_tuple(self):
+    def as_serializable(self):
         """
-        Exports intervals and length of ArrayInterval to a tuple. 
+        Exports intervals and length of ArrayInterval to a serializable object.
+        Reverting 
         Allows easy export of ArrayIntervals, e.g. into .json format. 
         >>> from IPython.lib.pretty import pprint
         >>> from paderbox.array.interval.core import ArrayInterval
         >>> ai = ArrayInterval.from_str('1:4, 5:20, 21:25', shape=50)
         >>> ai
         ArrayInterval("1:4, 5:20, 21:25", shape=(50,))
-        >>> ai.as_tuple
+        >>> ai.as_serializable
         ('1:4, 5:20, 21:25', (50,))
         """
         assert self.inverse_mode is False, 'Export of intervals as tuple is only valid for normal mode, not inverse mode!' 
         return self._intervals_as_str, self.shape
 
     @staticmethod
-    def from_tuple(array):
+    def from_serialized(array):
```

Comment:
The names `from_serialized` and `to_serializable` don't match. The serialized variant would be a string/bit stream and not a tuple, right?

Reply:
Do you think `from_serializable` would be a better name?

Reply:
It would at least match `to_serializable`. Without knowledge about what this function is used for, this name can be confusing. But, I don't have a better idea.

Reply:
I agree, `from_serializable` would be the counterpart to `to_serializable`.

 - serializable: Something that can be serialized
 - serialize(d): A stream that can be saved, e.g. `str`, `bytes`